ddtrace/internal/ci_visibility/encoder.py

```diff
@@ -199,7 +199,7 @@ def _convert_span(self, span, dd_origin):
         converted_span = {
             "test_session_id": int(span.get_tag(SESSION_ID) or "1"),
             "test_suite_id": int(span.get_tag(SUITE_ID) or "1"),
-            "files": json.loads(span.get_tag(COVERAGE_TAG_NAME))["files"],
+            "files": json.loads(span.get_tag(COVERAGE_TAG_NAME) or "{}")["files"],
```

Comment:
Doesn't this automatically cause a`KeyError` if we hit the `or`?

Reply:
Not that the original version is much better, though. I should probably just rewrite this.

Reply:
Glad you caught this, I had to add it because of a typing error:

```
ddtrace/internal/ci_visibility/encoder.py:202: error: Argument 1 to "loads" has incompatible type "Optional[str]"; expected "Union[str, bytes]"  [arg-type]
```

I see your point about the `KeyError` and changed it to cast the return value of `get_tag` to a string.